configutil/file_plugin_test.go

```diff
@@ -0,0 +1,168 @@
+package configutil
+
+import (
+	context "context"
```

Comment:
Why is this aliased?

Reply:
Probably `go-imports` or a similar tool doing something weird.